orchagent/mirrororch.cpp

```diff
@@ -748,7 +748,7 @@ bool MirrorOrch::setUnsetPortMirror(Port port,
     if (set)
     {
         port_attr.value.objlist.count = 1;
-        port_attr.value.objlist.list = (long unsigned int *)calloc(port_attr.value.objlist.count, sizeof(sai_object_id_t));
+        port_attr.value.objlist.list = reinterpret_cast<sai_object_id_t *>((void *)calloc(port_attr.value.objlist.count, sizeof(sai_object_id_t)));
```

Comment:
>calloc [](start = 83, length = 6)

`calloc` already returns a `void *`. Why you need to convert explicitly?

Reply:
reinterpret cast reports error if we don't explicitly type cast

Reply:
I mean why convert `calloc` to `void *`. It is already `void *`.

Reply:
removed explicit cast